docs/low-code-sql/data-tests/data-tests.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Data tests
+id: data-tests
+description: Data tests check the validity of your SQL
+sidebar_position: 8
+tags:
+  - concept
+  - testing
+  - sql
+---
+
+You can use data tests to ensure that your business data is generated reliably over time. As a data engineer, data analyst, or business user, you can run data tests so that you don’t have to manually check every Dataset every time you run a Job or model. The data test checks the validity of the SQL in your project.
+
+For example, find the following Test named `ref_int_orders_customers` which checks the validity of the SQL in the `HelloWorld_SQL` Project. In particular, the referential integrity check for orders and customers asserts that every `customer_id` entry in the `orders` table is present in the `customers` table.
+
+![Project test canvas](img/project-test-canvas.png)'
+
+This test starts with several **(1) models** from the `HelloWorld_SQL` Project, combines their data with a series of **(2) transformation** steps, and feeds the resulting table into the **(3) Data Test** Gem.
```

Comment:
If you use the (1), (2), (3) in the text, then I suggest adding the black icons to the image. That's one of the guidelines Maciej likes to use. 

Alternatively you can remove (1), (2), (3) from the text.

Reply:
Done.